data/brands/amenity/fast_food.json

```
@@ -495,7 +495,20 @@
       }
     },
     {
-      "displayName": "Barburrito",
+      "displayName": "Barburrito (CA)",
+      "id": "barburrito-e49d2e",
+      "locationSet": {"include": ["ca"]},
+      "tags": {
+        "amenity": "fast_food",
+        "brand": "Barburrito",
+        "brand:wikidata": "Q123765650",
```

This has been deleted, I've requested Q104844862 be [undeleted](https://www.wikidata.org/wiki/Topic:Xva33b6tdhkd5tyl).

If it gets undeleted we'll want to change it:
```suggestion
        "brand:wikidata": "Q104844862",
```

---

The original has been restored, and the new one is a redirect.